dom/apps/AppsUtils.jsm

```diff
@@ -956,7 +956,7 @@ ManifestHelper.prototype = {
       if (this._localeProp("start_url")) {
         return this._baseURI.resolve(this._localeProp("start_url") || "/");
       }
-      return this._baseURI.resolve(this._localeProp("launch_path") || "/");
```

Comment:
Current situation: calculator is broken and cannot be fixed because the bug is: their hosted manifest doens't have a `launch_path` attribute.

With this patch: calculator is fixed and we might break apps that don't have `launch_path` *and* rely on `/`

Alternate way to fix the issue: modify preload.py to not use the only version of the manifest but the one from the package. I chose not to do that because I assumed that using the online one was done on purpose (maybe for updating app?)